src/java.base/share/classes/java/time/OffsetDateTime.java

```diff
@@ -1801,7 +1802,11 @@ public long toEpochSecond() {
      * consistent with {@code equals()}.
      *
      * @param other  the other date-time to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value is less than zero if the {@code other} is before,
```

Comment:
As per `OffsetTime` this definition is wrong. The comparison order is not before/after.